packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -474,20 +494,19 @@ export class EngineReportingAgent<TContext = any> {
     // `await` the `signature` if it's a Promise, prior to putting it
     // on the stack of traces to deliver to the cloud.
     const cachedSignature = await this.signatureCache.get(cacheKey);
+    let graphqlValidationFailure = false;
```

Comment:
I don't think this variable adds anything over just writing `graphqlValidationFailure: true/false` in the appropriate places.

Reply:
Removed